make/devkit/Tools.gmk

```diff
@@ -63,20 +63,28 @@ ifeq ($(BASE_OS), OL)
     LINUX_VERSION := OL6.4
   endif
 else ifeq ($(BASE_OS), Fedora)
-  DEFAULT_OS_VERSION := 27
-  ifeq ($(BASE_OS_VERSION), )
-    BASE_OS_VERSION := $(DEFAULT_OS_VERSION)
-  endif
-  ifeq ($(filter x86_64 armhfp, $(ARCH)), )
-    FEDORA_TYPE := fedora-secondary
-  else
-    FEDORA_TYPE := fedora/linux
-  endif
-  ARCHIVED := $(shell [ $(BASE_OS_VERSION) -lt $(DEFAULT_OS_VERSION) ] && echo true)
-  ifeq ($(ARCHIVED),true)
-    BASE_URL := https://archives.fedoraproject.org/pub/archive/$(FEDORA_TYPE)/releases/$(BASE_OS_VERSION)/Everything/$(ARCH)/os/Packages/
+  ifeq ($(ARCH), riscv64)
+    DEFAULT_OS_VERSION := rawhide/68692
+    ifeq ($(BASE_OS_VERSION), )
+      BASE_OS_VERSION := $(DEFAULT_OS_VERSION)
+    endif
+    BASE_URL := http://fedora.riscv.rocks/repos-dist/$(BASE_OS_VERSION)/$(ARCH)/Packages/
```

Comment:
Hi,  thanks for adding devkit support for linux/riscv64.
I tried and it seems that I could not access this URL: http://fedora.riscv.rocks/repos-dist/rawhide/68692/riscv64/Packages/
Anything I missed here?

Reply:
Looks like the URL are accessible now again. Changes looks good to me.

Reply:
The varying availablility is a bit unnerving, as is the fact that only a single build of rawhide is present. :-(

Clearly, this is not an ideal way to get an environment to build riscv in. In fact, the way GHA does it by using Debian's debootstrap is probably much better. The only reason for doing it this way is that it fits the current devkit model and thus do not require a complete rewrite of the devkit make files.